handlers/handler.go

```diff
@@ -0,0 +1,66 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"strings"
+)
+
+type IndexdInfo struct {
+	URL      string `url`
+	Username string `username`
+	Password string `password`
+}
+
+func getIndexServiceInfo() (*IndexdInfo, error) {
+	indexdInfo := new(IndexdInfo)
+	if err := json.Unmarshal([]byte(os.Getenv("IMAGE_CONFIG")), indexdInfo); err != nil {
```

Comment:
`IMAGE_CONFIG` seems like a strange name for this? also could you add some documentation on this variable? I feel like a config file probably is more convenient but this works fine

Reply:
Agree! Changed in ba795498eb1e0537bae5bb2abb54fe15b7f7e6c5